.env.example

```diff
@@ -5,4 +5,11 @@ BUILD_THREADS=1
 # Updates NET_VERSION in CMakeVariables.txt
 BUILD_VERSION=171022
 # make sure this is a long random string
-ACCOUNT_MANAGER_SECRET=
\ No newline at end of file
+ACCOUNT_MANAGER_SECRET=
+# Should be the externally facing IP of your server host
+EXTERNAL_IP=localhost
+# Database values
+MARIADB_USER=SECRET_VALUE_CHANGE_ME
```

Comment:
I would just set this by default to darkflame or something not SECRET_VALUE_CHANGE_ME. The same with MARIADB_DATABASE.

Reply:
Done